src/tool/hpcrun/ompt/ompt-interface.c

```diff
@@ -699,6 +698,41 @@ ompt_idle_blame_shift_request()
   ompt_register_idle_metrics();
 }
 
+//*****************************************************************************
+// map opid operations
+//*****************************************************************************
+
+// Records the (target id —> cct_node *) mapping in a thread local variable
+void
+hpcrun_opid_map_record_target(int target_id,
+                              cct_node_t *node)
+{
+  assert(target_id != 0);
+  ompt_region_map_insert((uint64_t) target_id, node);
```

Comment:
I notice there's already a region_map structure, so I use it to map `target_id` to `cct_node_t`